src/webrtc/groupCall.ts

```diff
@@ -408,6 +414,12 @@ export class GroupCall extends EventEmitter {
 
         // set a timer for the maximum transmit time on PTT calls
         if (this.isPtt) {
+            // if anoher user is currently unmuted, we can't unmute
+            if (!muted && this.userMediaFeeds.some(f => !f.isAudioMuted())) {
+                throw new OtherUserSpeakingError();
+            }
```

Comment:
This seems susceptible to ping response times? If multiple people unmute at the same time, they all would be allowed.